Tests/L0/ServiceFabricDeploy/CertDeploy.ps1

```diff
@@ -0,0 +1,57 @@
x/GB4hNQmLkYvK52kZQ6PWC6bTRx3msEAGSc1ZCs7JU//5bhLFlfZaoZkAfR9DvDefCDQc/VqfIjTP+6V/aQvgS8I7zkzQ21kWEhXvMf7glAzbC839tMVPHy7+Jyrf86FI4ye3UCEwNeSxC3eFuS9skL6TQi/peTNhTl2P6MlX4/iIxDCel0E5VFIY2vQYUy4zhCabZqLP1tOCEzNNCmpGAdwCLhNdXjICBdG9TVmaYJazJqraIqD+RG6x+RDd2OJmB687Fq13Jr2P0YhDE1dDnW3Uf9KB7e8XDWVfnjnpdUB7tUE10TcRY7eAgRRy2wO5EOOB8Bl6w7zxT3x6FtLE3+SXmsnK5E+SovZ04LcLiKQfbdDFRJjWbtkiNgkF1WmansnT3yAegWL2L02M2s+AwH/UWgjqUit+53vigBl6j5j3ihvI4QOO7F4Y5m/2a2XKH5tVlpdHmukiifvaMDQdrMBbSNWUigQjSh8kC/fz54Kdx/Fkfk6FZLo9Z0JcQuMSfgbmyhX+NKzm21ASXnnBIzx4zBCb7IFqPy1+cWPmaPWxVk423a5KpGUALEtyWem9QIwQ0aMPo68dOWoRb18Peo5foyyFlb1oQMUG3q7rj7NcZARDp9ugahIrBcdm5nCLmbLl9tbs9G9hk2+IGAelkD4aFknBC8iPyCSn9pKXA7A0Q+p7PBiaHzq5T9CTUj2VZJZ848m4dsFaEZ0M2Ym0aj+/E7xEbb0t8UKk3U2yDPf/aQKBlddlQjLQY6CCkm8n1w+Lx8q81YR56lBrQtNjKFcqaP9mZ1CNlKwarDUiLIjKbchj0GXrtKvDwNjtetQ3oIweJfeOKjMX9NYknyfTKNnFR70belYf57fFXe/BfpjbieIR8FYSEUFW6iARdLC368v2ECGK68Gz/1xhY+6MA/bFvaDsoRopZbdWM2OUSgLzlFffHqxFlzzcB51ez9wvJlfJ2mkLaHZbDOetM3ua7Ehk8QR8WBeD5JheVQdPBr0VH87J4EcHokskpGO53KGIL/7xLm4RS7DjlOJfBGwe+U4+lSI37ks0VhPQgulfHUFWA2GEowc/npRgXgvU39zOObaVG3a3rK67dghUUIm/7zZE8V21aEGWSFcvmppO47pc7x67IRUcDXppcIJROZRkMGYnCQq6ZDH/sEpd2ia6fJVaz7vE7Vm/GRvGBOMxhtJ9s6yFVtLDN2ri/1BF83sDn5gxfDSBTUc0OBl7Hl+EfOUVPnfdQmoSpMLqS4SZVWKrP/Sf7+vcsBl+XMQcenHmHrTuLC+z+7ZV+x2XrHsDwxXorLjGB0zATBgkqhkiG9w0BCRUxBgQEAQAAADBdBgkqhkiG9w0BCRQxUB5OAHQAZQAtAGYAOQA4AGEAZgAzADUAOAAtAGIAZgBkADAALQA0ADkAOQA5AC0AOQBhADcANAAtAGYAZgBlADQANQBlADkAZQBkADMAYgBiMF0GCSsGAQQBgjcRATFQHk4ATQBpAGMAcgBvAHMAbwBmAHQAIABTAG8AZgB0AHcAYQByAGUAIABLAGUAeQAgAFMAdABvAHIAYQBnAGUAIABQAHIAbwB2AGkAZABlAHIwggPbBgkqhkiG9w0BBwGgggPMBIIDyDCCA8QwggPABgsqhkiG9w0BDAoBA6CCAzQwggMwBgoqhkiG9w0BCRYBoIIDIASCAxwwggMYMIICAKADAgECAhA+xB+Ws8MWtUNQG41xK066MA0GCSqGSIb3DQEBCwUAMBQxEjAQBgNVBAMMCWxvY2FsaG9zdDAeFw0xNjA3MDcxODUyMjFaFw0xNzA3MDcxOTEyMjFaMBQxEjAQBgNVBAMMCWxvY2FsaG9zdDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAO+Vpe8+85bhbp9qNby/cBuDstSKBxn6WBFLGd5oKuOGMyKl6ddgklXBS5V5Co8jO3fgXOlDVc+bKIX2jGwSWZODjWhdUP0f+Wq7OvpBZ9p+duUUv/XgdQcXsuEn6VfsJHczBI9+D8+LYH7NvBqXI6PZwruaYEVprCPXNYvVtG4XwUOFops8T+IOXzxoaGXFsK3tQ9QzTYa1AJLP6UzK0HTsFV5if9f7cxin5JUVo2NdCVHkJykLh2+cePGhqLlEuZhVnnSlAdpDqt6ENTq05Vo0d0hSFIujL8jwrRzzHqZlLAIQyCddlb7+EtgtMSJE0/ofa3MFHuhMiDBmJBGrM78CAwEAAaNmMGQwDgYDVR0PAQH/BAQDAgWgMB0GA1UdJQQWMBQGCCsGAQUFBwMCBggrBgEFBQcDATAUBgNVHREEDTALgglsb2NhbGhvc3QwHQYDVR0OBBYEFCd40q3frUHTSKMTZbR7MyBNME/FMA0GCSqGSIb3DQEBCwUAA4IBAQCOTIACyJRBTzaEILCsRNuFmDAVxaGwF7iOMptQdmhc8As3AssHvy+ZrBZ6adN4c1HprGLpUAiiU09gZWTcGR2I6CvoyQRXXRmbpV3alHD0p4QuQWuvxMghN72QCnOqFX6ZlTt6XZJ2UUeNuFL2oop/nXbaxr6HyIQVW4HpiNvCAuF4VWr/zEDCHLDYEYbaWMDxN0+qZa6thfGMphis2OOsyS21ebI2Iy/hPs3ViYzehZML0LdNQ85Kgtpn4Wg93chW/HhM7y74O1ZktvqXbUCcRktQxsOmYU3gc/tjEkF5+IC4cOOKvKZTIeaF8pdMOgQLN4YTOmSFz0dxTLYL6DjWMXkwEwYJKoZIhvcNAQkVMQYEBAEAAAAwYgYKKwYBBAGCNxEDRzFUBFJkAGQAbQB0AGgAYQBsAG0AYQBuADAANQA2AC4AcgBlAGQAbQBvAG4AZAAuAGMAbwByAHAALgBtAGkAYwByAG8AcwBvAGYAdAAuAGMAbwBtAAAAMDswHzAHBgUrDgMCGgQUP8Xsekjrz1Wg4zWW4tl7ZTfbwKEEFOZHn1JKi8IT9sVz6KSGUJYAbApOAgIH0A=="
```

Comment:
I hope you're not sharing an important secret here :)

Is this going to import the cert into my machine when the L0 test runs?